modules/@apostrophecms/schema/ui/apos/components/AposInputSlug.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <AposInputWrapper
+    :modifiers="modifiers" :field="field"
+    :error="error" :uid="uid"
+  >
+    <template #body>
+      <div class="apos-input-wrapper">
+        <textarea
+          :class="classes"
+          v-if="field.textarea" rows="5"
+          v-model="next" :placeholder="field.placeholder"
+          @keydown.enter="$emit('return')"
+          :disabled="field.disabled" :required="field.required"
+          :id="uid" :tabindex="tabindex"
+        />
+        <input
+          v-else :class="classes"
+          v-model="next" :type="type"
+          :placeholder="field.placeholder"
+          @keydown.enter="$emit('return')"
+          :disabled="field.disabled" :required="field.required"
+          :id="uid" :tabindex="tabindex"
+        >
+        <component
+          v-if="icon"
+          :size="iconSize"
+          class="apos-input-icon"
+          :is="icon"
+        />
+      </div>
+    </template>
+  </AposInputWrapper>
+</template>
+
+<script>
+// NOTE: This is a temporary component, copying AposInputString. Base modules
```

Comment:
See here. I kept getting errors, so I stubbed this in.

Reply:
Makes sense.